packages/expo-sqlite/android/src/main/cpp/NativeDatabaseBinding.cpp

```diff
@@ -48,6 +52,17 @@ void NativeDatabaseBinding::registerNatives() {
 
 int NativeDatabaseBinding::sqlite3_changes() { return ::exsqlite3_changes(db); }
 
+long NativeDatabaseBinding::sqlite3_next_stmt(long stmt) {
+  exsqlite3_stmt* nativeStmt = reinterpret_cast<exsqlite3_stmt*>(stmt);
+  exsqlite3_stmt* next = ::exsqlite3_next_stmt(db, nativeStmt);
+  return reinterpret_cast<jlong>(next);
```

Comment:
also i'm little hesitant to introduce the opaque jlong for pointers. 
thinking out loud, maybe we can just introduce a `sqlite3_finalize_all_statement()` function like
```cpp
int NativeDatabaseBinding::sqlite3_finalize_all_statement() {
  ::exsqlite3_stmt *stmt = ::exsqlite3_next_stmt(db, nullptr); 
  while (stmt) {
    ::exsqlite3_stmt *nextStmt = ::exsqlite3_next_stmt(db, stmt);
    ::exsqlite3_finalize(stmt);
    stmt = nextStmt;
  }
}
```

then we don't have to keep back-and-forth between android jni

Reply:
yes, please help to do this on your pr. if you have any questions, i'm here with you.